synfig-studio/src/gui/states/state_normal.cpp

```diff
@@ -355,11 +354,7 @@ DuckDrag_Combo::begin_duck_drag(Duckmatic* duckmatic, const synfig::Vector& offs
 
 	bad_drag=false;
 
-		drag_offset=duckmatic->find_duck(offset)->get_trans_point();
-
-		//snap=drag_offset-duckmatic->snap_point_to_grid(drag_offset);
-		//snap=offset-drag_offset_;
-		snap=Vector(0,0);
+	drag_offset=duckmatic->find_duck(offset)->get_trans_point();
```

Comment:
Take this opportunity to surround `=` sign with spaces XD

Reply:
oh yes sure :)